GoogleSignIn/Sources/GIDSignInInternalOptions.h

```diff
@@ -77,6 +80,7 @@ NS_ASSUME_NONNULL_BEGIN
                                       loginHint:(nullable NSString *)loginHint
                                   addScopesFlow:(BOOL)addScopesFlow
                                          scopes:(nullable NSArray *)scopes
+                                          nonce:(nullable NSString *)nonce
```

Comment:
Let's add a method so that we do not break existing API and force major version release.

Reply:
@mdmathias this is not a public api, afaict: https://github.com/google/GoogleSignIn-iOS/blob/b0a724e1a8e12cfbe027d8f844a1e6f80f17532f/GoogleSignIn.podspec#L24

plus it's called "internal"

Reply:
Apologies for my confusion. IIRC, I was thinking this is how you were trying to pass in the `nonce`, but I realize now it coming from the sign in methods.